### ISSUE TITLE
Draft Packages

### ISSUE THREAD

**Issue by StuartFeldt** [StuartFeldt/Hashtags#61]
- Do we only have three? 
- Standard, Plus, Pro?  
- Are these good names?
- What should they be priced at?
- What features are included in each?

**Comment by a1exis** [comment]
OK. I think that three packages are great to start. I don't know about the names--i think we should nix the 'pro' because usability studies show that people automatically have an aversion to that because it's associated with 'professional.'  'Premium' tends to work better.  

I like the direction that you're heading in for packages. I was thinking we might also be able to have an "add-on" of a PDF/book of some sort where all of the tweets and photos are included in a timeline. That can be an extra $50 or something. Would this be possible from a functionality perspective? If it is, and you can send me a raw file of what it might look like, i can explore some options on how we could make it pretty.

**Comment by StuartFeldt** [comment]
Yea, a PDF seems possible.  What could also be cool would be to make a video representation and upload straight to YouTube.  That would be damn near impossible, though worth investigating.

HTML to PDF seems possible, I'll create an issue to possibly develop it: https://code.google.com/p/dompdf/

**Comment by a1exis** [comment]
love the idea of a youtube video...don't know why i didn't think about that.  Basically just something that lets it exist outside of the 2-week (or whatever) span.

as i work through the SWOT, pricing will hopefully figure itself out.  For now, i like where you're at.

**Comment by StuartFeldt** [comment]
So would they be able to purchase the video later, like after the two week period?  Or would they have to commit while signing up?

The reason I ask is I don't want the free version to force a user to sign up/register.  It's such a pain to do, and the standard version should be as absolutely simple and easy to create as possible.  

That said, we may have to at least make users specify an "admin" password for editing their sites.  Obviously for the higher packages, they would need to sign in and I guess see some kind of panel for changing hashtags, blocking users, changing themes, etc.  Maybe similar to http://hashtagmyevent.com/admin   (user: admin, password: adminpass)

**Comment by a1exis** [comment]
I agree with what you're thinking--its best to avoid creating an account, because tons of people drop off in the process. about your 'admin' password thing--that site isn't working for me? but if we don't want them to create an account, we could have a password automatically assigned to them (maybe their last name?) 

I think they definitely should be able to purchase the video later, but i'm not sure how much later (that depends on how much work it is for you to do, how much bandwith, etc.) 

I could write up a promotional email about the video, and we could send it out the morning after the event. "Want yesterday to last forever?Purchase a custom YouTube video in timeline mode, so you can view it from beginning to end--whenever you want." (OK that was bad, but you know what i mean.)

**Comment by StuartFeldt** [comment]
Yea, the entire site is broken for some reason...  I'll try to look at this tonight.  http://dev.hashtagmyevent.com should be working

**Comment by StuartFeldt** [comment]
And I like the email thing to auto-send out after the event.
